### ISSUE TITLE
Number of Classes in FieldsOfTheWorld Dataset

### ISSUE THREAD

**Issue by nilsleh** [microsoft/torchgeo#2696]
### Description

I am iterating through the 3-class version of the FieldsOfTheWorldDataset and noticing that for the country of brazil and sometimes india the max label value is 3, so others return [0, 1, 2] but for brazil [1, 2, 3] are returned as mask labels. 

Do you perhaps have any insights @calebrob6 ?

### Steps to reproduce

```python

ds = FieldsOfTheWorld(root="FieldsOfTheWorld", target='3-class', countries=CC_BY_COUNTRIES)

dl = DataLoader(ds, batch_size=64, num_workers=8)

for batch in tqdm(dl):
    if batch["mask"].max() > 2:
        print("Max value in mask is greater than 2")
```

or check these two examples directly:

```python
path = "FieldsOfTheWorld/brazil/label_masks/semantic_3class/g3_0000011264-0000001536.tif"
path = "FieldsOfTheWorld/india/label_masks/semantic_3class/g1_1799.tif"

with rasterio.open(path) as src:
    mask = src.read(1)
    print(mask.max())
    print(np.unique(mask))

```

### Version

'0.7.0.dev0'

**Comment by calebrob6** [comment]
I think this is a documentation issue. Confirming now, but Brazil, Kenya, India, and Rwanda all have "presence only" labels so I think we set the background class to "3" (as an "unlabeled" class) in those cases instead of 0 ("definitely not a field" class).

**Comment by calebrob6** [comment]
Confirmed, below shows the `country, number of samples in the training set that contain a "3" in the mask, total number of samples in the training set, fraction: 
```
austria,0,5303,0.0000
belgium,0,1554,0.0000
brazil,1283,1289,0.9953
cambodia,0,245,0.0000
corsica,0,1974,0.0000
croatia,0,2778,0.0000
denmark,0,2868,0.0000
estonia,0,5348,0.0000
finland,0,4527,0.0000
france,0,2773,0.0000
germany,0,306,0.0000
india,1261,1261,1.0000
kenya,316,316,1.0000
latvia,0,5529,0.0000
lithuania,0,4208,0.0000
luxembourg,0,643,0.0000
netherlands,0,3110,0.0000
portugal,0,47,0.0000
rwanda,57,57,1.0000
slovakia,0,3275,0.0000
slovenia,0,1733,0.0000
south_africa,0,590,0.0000
spain,0,2015,0.0000
sweden,0,3802,0.0000
vietnam,0,228,0.0000
```

**Comment by nilsleh** [comment]
Okay, so the loss function needs to have an `ignore_index=3` for the loss function like [here](https://github.com/fieldsoftheworld/ftw-baselines/blob/74a8f1605ab0ac7739ac61b47d4c0b30d6113b63/src/ftw_cli/model.py#L88), maybe that should be the documentation update?

**Comment by calebrob6** [comment]
I updated docstring to clarify that 3 is unlabeled -- in our paper we train ignoring "3" and also mapping it back to 0 and ge mixed results
